src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2912,15 +2959,14 @@ public void execute() throws GitException, InterruptedException {
         };
     }
 
-
```

Comment:
Please don't cleanup whitespace as part of a pull request.  Merge conflicts increase dramatically with white space changes

Reply:
fixed